plugins/email_service/app/controllers/email_service/domain_verifications_controller.rb

```diff
@@ -1,161 +1,216 @@
-      else
-        render :new and return
+
+      @dkim_types = ::EmailService::VerifiedDomain.dkim_types
+      @rsa_key_length = ::EmailService::VerifiedDomain.key_length
+      
+      # Rails.logger.debug "\n[email_service][domain_verifications_controller][create]\n"
+      begin
+        # domain_verification_params
+        @verified_domain = domain_verification_form(domain_verification_params)
+
+        # Rails.logger.debug "\n ************ domain_verification_params.inspect #{domain_verification_params.inspect} ************ \n "
+        # Rails.logger.debug "\n ************ @verified_domain.inspect #{ @verified_domain.inspect } ************ \n "
+        # Rails.logger.debug "\n ************ @verified_domain.valid? #{ @verified_domain.valid? } ************ \n "
+        # # https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/SESV2/Types/DkimSigningAttributes.html
+
+        # # TODO tags
+        # # Capture in user input field with JavaScript dynamic input field population
+        # # Aggregate in to a hidden text field and send over to controller to process.
+        tags = [{ key: "color", value: "red"}, { key: "type", value: "not_junk_mail"}]
+        
+        @verified_domain.tags = tags
+
+        if @verified_domain.valid?
+          msg = process_domain_verification(@verified_domain)
+          # Rails.logger.debug "\n ****** domain_verification_values.inspect #{@verified_domain.inspect} \n @verified_domain.nil? : STATUS MESSAGE : #{msg} ************ \n"
+          if msg == "success"
+            flash[:info] = DOMIAN_VERIFICATION_INITIATED
+          end
+          redirect_to plugin('email_service').domain_verifications_path and return
+        else
+          render :new and return
+        end
+        rescue Elektron::Errors::ApiResponse => e
+          error = "#{I18n.t('email_service.errors.domain_verification_create_error')} #{e.message}"
+          Rails.logger.error error
+          flash[:error] = error
+        rescue Exception => e
+          error = "#{I18n.t('email_service.errors.domain_verification_create_error')} #{e.message}"
+          Rails.logger.error error
+          flash[:error] = error
+        redirect_to plugin('email_service').domain_verifications_path
       end
-      rescue Elektron::Errors::ApiResponse => e
-        flash[:error] = "Status Code: #{e.code} : Error: #{e.message}"
-      rescue Exception => e
-        flash[:error] = "Status Code: 500 : Error: #{e.message}"
 
     end
 
     def show
-      @verified_identity = find_verified_identity_by_name(params[:identity], "Domain")
+      # Rails.logger.debug "\n[email_service][domain_verifications_controller][show]\n"
+      begin
+        @verified_identity = find_verified_identity_by_name(params[:identity_name], "DOMAIN")
+        # Rails.logger.debug "\n [DomainVerificationsController][find_verified_identity_by_name]:[params.inspect: \n [#{params.inspect}] \n"
```

Comment:
Do you really need to keep commented all this rails loggers?

Reply:
removed.

Reply:
removed.